staging/src/k8s.io/client-go/examples/third-party-resources/apis/tpr/v1/register.go

```diff
@@ -24,28 +24,25 @@ import (
 
 var (
 	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
-	AddToScheme   = SchemeBuilder.AddToScheme
 )
 
-// GroupName is the group name use in this package
+// AddToScheme applies all the functions stored in the SchemeBuilder value to a scheme.
+func AddToScheme(s *runtime.Scheme) error {
```

Comment:
I concede that I'm thinking defensively here, like I do when writing code in my own projects, and maybe the Kubernetes libraries don't bother with that kind of defensive posture. I favor exporting as little as possible from a package, and limiting callers' abilities to deliberately or inadvertently break the intended behavior of the package. Keep that perspective in mind when considering these suggestions.